charts/prom-exporter/templates/secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
starting from 
https://github.com/lucidworks/terraform-google-gke/releases/tag/0.4.7
tf module cluster default sa should be able to pull docker images from artifact registry without extra auth so this secret is redundant assuming with will be a part of the latest deployments 
the same applies to `imagePullSecrets`